src/EPR.PRN.Backend.Data/Repositories/RecyclingTargetRepository.cs

```diff
@@ -2,20 +2,12 @@
 using EPR.PRN.Backend.Data.Interfaces;
 using Microsoft.EntityFrameworkCore;
 
-namespace EPR.PRN.Backend.Data.Repositories
+namespace EPR.PRN.Backend.Data.Repositories;
+
+public class RecyclingTargetRepository(EprContext context) : IRecyclingTargetRepository
```

Comment:
same here with primary constructor

Reply:
Yes, that's suggestion from Visual Studio / GitHub. Hence, changed